projects/kubernetes/kubernetes/1-20/checksums

```diff
@@ -0,0 +1,19 @@
+1526a6d976277207ed4c3398b457031aa3908f671c1871882eaee7a64098a2b6  _output/1-19/bin/windows/amd64/kube-proxy.exe
```

Comment:
how are these checksums calculated?
Would appreciate instructions in the readme.md about the process for updating relevant files.